_gsocproposals/2019/proposal_WLCGansible.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Python Components for the SIMPLE Grid Framework
+layout: gsoc_proposal
+project: WLCG
+year: 2019
+organization:
+  - CERN
+---
+
+## Description
```

Comment:
The 2 projects have identical description. The general part can be indeed common, but the second part is maybe more useful to describe the specific task for one proposal or another.

Reply:
Indeed. I have updated the title and description of the Ansible project. Thanks for pointing this out.